src/mame/msx/msx_kanji12.h

```diff
@@ -0,0 +1,40 @@
+// license:BSD-3-Clause
+// copyright-holders:Wilbert Pol
+#ifndef MAME_MSX_MSX_KANJI12_H
+#define MAME_MSX_MSX_KANJI12_H
+
+
+#include "msx_switched.h"
```

Comment:
This should have the `#pragma once`, too.

Reply:
pragma added